.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+
+go:
+  - 1.4
+  - tip
+
+env:
+  - GOTAGS=restapi
+  #- GOTAGS=powershell
+
+install:
+  # trick to make go find packer-azure packages under MSOpenTech
+  - ln -s paulmey ../../MSOpenTech

```

Comment:
Hey, it worked for me, regardless if that's how symlinks work... :-)
Anyways, it shouldn't be in the pull request. It was necessary to have travis build changes in my fork (thanks to go having hardcoded paths to specific github repo's)